backend/tests/auth/test_auth.py

```diff
@@ -0,0 +1,44 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+from src.main import initialize_backend_application
```

Comment:
If we run this with all the components are running. It should be useful. However, in our test, we don't want to deploy pg and vector db. So, we need to use like https://github.com/tiangolo/fastapi/discussions/8127.